e2e/teams.loggedin.spec.ts

```diff
@@ -0,0 +1,195 @@
+import { Page, expect, test } from '@playwright/test'
+import { v4 as uuidv4 } from 'uuid'
+import { ADMIN_STORAGE_STATE, USER_STORAGE_STATE } from '../playwright.config'
+import { getLocalStorage } from './helpers/localStorage'
+import {
+  deleteTeamInvitationMessage,
+  deleteTeamInviteAcceptanceMessage,
+} from './helpers/messages'
+import {
+  USER_NAME_LOCALSTORAGE_KEY,
+  getAccessTokenFromCookie,
+  getAdminUserCredentials,
+  getUserIdFromLocalStorage,
+  goToDashboard,
+} from './helpers/testUser'
+
+const TEAM_NAME = 'swc-e2e-team-' + uuidv4()
+const INVITATION_MESSAGE = 'swc e2e test team invitation' + uuidv4()
+
+let adminPage: Page
+let userPage: Page
+
+test.describe('Teams', () => {
+  test.beforeAll(async ({ browser }) => {
+    adminPage = await browser.newPage({
+      storageState: ADMIN_STORAGE_STATE,
+    })
+    userPage = await browser.newPage({
+      storageState: USER_STORAGE_STATE,
+    })
+  })
```

Comment:
Could refactor to create [authenticated page fixtures](https://sagebionetworks.jira.com/jira/software/c/projects/SWC/issues/SWC-6528), which would remove the need to create new pages in the `beforeAll` hook

Reply:
Thanks for creating the issue to track